### PR TITLE
Tags Mismatch 

### PR DESCRIPTION
**Pull request by tvaron3**
# Description
Prometheus expects that for every meter every possible tag is present. This pr adds `RequestStatusCode` and `PartitionKeyRangeId` tags to all query plan meters to adhere to the prometheus expectations. These tags will just be filled in with `NONE` string following the convention we follow in other tags where a value is missing. The `RegionName` tags in operation meters will also follow this convention now.

### CODE REVIEW

**Review by jeet1995 — APPROVED** [review]
LGTM - thanks!

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java [hunk 1]
```diff
@@ -373,13 +373,11 @@ private static Tags createOperationTags(
...
 
-        if (contactedRegions != null &&
-            contactedRegions.size() > 0 &&
-            metricTagNames.contains(TagName.RegionName)) {
-
+        if (metricTagNames.contains(TagName.RegionName)) {
             effectiveTags.add(Tag.of(
                 TagName.RegionName.toString(),
-                String.join(", ", contactedRegions)
+                contactedRegions != null && contactedRegions.size() > 0
```
Comment (jeet1995) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java:379]:
```suggestion
                contactedRegions != null && contactedRegions.isEmpty()
```

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ClientMetricsTest.java [hunk 2]
```diff
@@ -1420,6 +1424,7 @@ public void metricConfigsThroughSystemProperty() {
                 .getCosmosClientTelemetryConfigAccessor()
```
Comment (tvaron3) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ClientMetricsTest.java:1424]: Add changelog

**Pull request merged**